unstructured/file_utils/ndjson.py

```diff
@@ -0,0 +1,24 @@
+import json
```

Comment:
In my view every module should have a docstring answering the question "What would I want to know about this module if I were encountering for the first time?"

I can think of a few topics:
- Why do we need `ndjson`?
- Is it something we partition?
- Who's using it now?
- How robust is this?
- Why didn't we use a standard library for this?

Perhaps some others will occur to you.

It's up to each of us to keep the code-base understandable for all those who end up here and using prose to expose the non-obvious aspects is an important part of that in my view.